master/internal/config/config.go

```diff
@@ -461,6 +481,28 @@ type TLSConfig struct {
 	Key  string `json:"key"`
 }
 
+// Validate implements the check.Validatable interface for the TokenConfig.
+func (t *TokenConfig) Validate() []error {
+	var errs []error
+	if t.MaxLifespanDays < 1 && t.MaxLifespanDays != InfiniteTokenLifespan {
+		errs = append(
+			errs,
+			errors.New("Max Token Lifespan must be greater than 0 days, unless set to -1 for infinite lifespan"),
```

Comment:
for better or worse, all our other errors are start with a lower case (excluding when they start with initialisms) so let's keep it consistent.